src/client/player/components.rs

```diff
@@ -5,3 +5,9 @@ pub struct HighlightCube;
 
 #[derive(Component, Default)]
 pub struct Player;
+
+#[derive(Component)]
+pub struct PlayerCamera;
+
+#[derive(Component)]
```

Comment:
I prefer whitelist over blacklist for raycasting. It gets messy with the UI and it is a better separation of concern.

The UI shouldn't be concerned by the player raycasting selection.